DESCRIPTION

```diff
@@ -32,6 +32,7 @@ Imports:
     purrr,
```

Comment:
@thomas-neitmann  still receiving rmd fail despite adding hms to decription file here